### PR TITLE
Fixes crosses

### PR DESCRIPTION
**Pull request by UndeadNappist**
-Makes it so unbuckling from crosses actually works.

-Makes it so they don't arbitrarily damage you just for being strung up on one.

-Makes it so you aren't rotated upside-down when you get buckled to a cross (Really?)

-Plays the correct noises now.

-Does the correct animations now.

-Reduces damage from being pulled and put onto a cross.

-Better flavortext

-Better, less buggy interactions

-Very minorly reduces damage from being on a kitchen spike so you don't just resist free and then instantly die of bloodloss.

-Cleans up some code.

There's probably a way to rewrite this chunk of code so there's less copypasting.

##
tl;dr the entire cross code was busted and not being used and it was defaulting to parent kitchenspike code

@rhicora

### CODE REVIEW

**Comment by rhicora** [comment]
How tested is this?

**Comment by UndeadNappist** [comment]
E x t e n s i v e l y.
